docs/api/naj-quick-reference.md

```diff
@@ -71,7 +71,7 @@ const homedir = require("os").homedir();
 
 const ACCOUNT_ID = "near-example.testnet";  // NEAR account tied to the keyPair
 const NETWORK_ID = "testnet";
-const KEY_PATH = "/.near-credentials/testnet/near-example.testnet.json";
+const KEY_PATH = `/.near-credentials/${NETWORK_ID}/${ACCOUNT_ID}.json`;
```

Comment:
I'm wondering if it would make sense to have the `KEY_PATH` be something unique that would indicate the special use case of pointing directly to a file. If a user was using the default keyPair for an account in the default directory of that network... they should just use the "Using Credentials Directory" example.

Maybe we can change this to highlight that you would only use this if you wanted to use a special key pair to sign this transaction... maybe a special function access key or something of the like... What do you think @think-in-universe ?

Something like:



```suggestion
// path to your custom keyPair location (ex. function access key for example account)
const KEY_PATH = '/.near-credentials/near-example-testnet/get_token_price.json';
```

Reply:
good idea. we do have developers who want to test with function call key in this way.